modules/tinymce/changelog.txt

```diff
@@ -1,4 +1,5 @@
 Version 5.6.0 (TBD)
+    Fixed the root document able to be scrolled when a dialog is open and rendered inside a shadow root #TINY-6363
```

Comment:
This reads a little odd.

Maybe

```suggestion
    Fixed the root document being able to scroll when a dialog is open and rendered inside a shadow root #TINY-6363
```

Reply:
`being` is in our list of words we're not allowed to use in changelogs/docs, so can't do that sorry.

Reply:
This might not be any better, but would `Fixed the root document incorrectly scrolling when a dialog...` work?

Reply:
That's nicer to read, but it has a different meaning, so I can't change to that sorry. That implies it scrolls without user interaction.

Reply:
Doesn't have the word "Fixed" but what about "Prevented scrolling in the root document when a dialog is open and rendered inside a shadow root"

Reply:
Needs to start with `Fixed` as you already know :stuck_out_tongue: and between that requirement and our banned words list, this is just really hard to come up with something nice.

Reply:
```suggestion
    Fixed an issue where the root document could be scrolled while an editor dialog was open inside a shadow root #TINY-6363
```